### PR TITLE
fix the command output

### PR DESCRIPTION
**Pull request by tanshanshan**
I have verified on version v1.11

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
> Help editing and submitting pull requests:  https://deploy-preview-9510--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#submit-a-pull-request.
> Help choosing which branch to use, see
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle-pr-reviews

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ To keep `kube-dns`, pass `--feature-gates=CoreDNS=false` to `kubeadm upgrade app
 
     ```shell
     kubectl drain ip-172-31-85-18
-    node "ip-172-31-85-18" cordoned
+    node/ip-172-31-85-18 cordoned
```
Comment (xiangpengzhao) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md]: @tanshanshan have you verified that the output is in this format against a 1.10.x cluster? I'm not sure about this.

Comment (xiangpengzhao) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md]: It describes an upgrade process and I think here the command is executed before upgrading.

Comment (tanshanshan) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md]: no, starting from 1.11

**Pull request closed without merging**